examples/kafka-rebalance/kafka-rebalance.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
Is the `kafka-rebalance` really what the users will be looking for?I'm not sure it says enougn. Why not just `cruide-control`. If nothing else I would cut the `kafka-` part since almost everything there is `kafka-`. But I think `cruide-control` or for example `cluster-rebalancing` would be more eye catching. A user who donwloads the examples and will see it will immediately know that we have something in this area.

Reply:
I was following the same pattern as for example user and topic ... we have `kafka-topic.yaml` and `kafka-user.yaml` example files, so `kafka-rebalance.yaml` seems to be fine to me.

Reply:
The comment was more about the directory than the file it self. I think the file name is fine.

Reply:
Yeah I agree, changing the folder name is a good idea.